### PR TITLE
brandon/url_and_index_test

### PR DESCRIPTION
**Pull request by ahkuma**

### CODE REVIEW

File: src/javascript/app_2/Utils/URL/__tests__/url_helper.spec.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        it('should return an object', () => {
+            expect(typeof params).to.eql('object');
+        });
+    });
+
+    describe('.setQueryParam', () => {
+        let params;
+        beforeEach('Setting up params', () => {
+            params = {
+                'currency': 'MYR',
```
[the remark this answers is not in the document]
Comment (ahkuma) [reply src/javascript/app_2/Utils/URL/__tests__/url_helper.spec.js]: MYR isn't a valid currency?

[the remark this answers is not in the document]
Comment (negarn) [reply src/javascript/app_2/Utils/URL/__tests__/url_helper.spec.js]: we don't offer it, I mean a valid binary currency

[the remark this answers is not in the document]
Comment (ahkuma) [reply src/javascript/app_2/Utils/URL/__tests__/url_helper.spec.js]: ah

File: src/javascript/app_2/Stores/Modules/Statement/Helpers/__tests__/format_response.spec.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        transaction_time: 123456789,
+        transaction_id  : 1234,
+        payout          : 1000,
+        amount          : 2000,
+        balance_after   : 3000,
+        longcode        : 'test \n test \n test',
+        contract_id     : 1234,
+        app_id          : 1234,
+    };
+    const currency = 'USD';
```
[the remark this answers is not in the document]
Comment (ahkuma) [reply src/javascript/app_2/Stores/Modules/Statement/Helpers/__tests__/format_response.spec.js]: we only have fiat and crypto right?

[the remark this answers is not in the document]
Comment (ahkuma) [reply src/javascript/app_2/Stores/Modules/Statement/Helpers/__tests__/format_response.spec.js]: actually I don't think we need to test other currencies as the function responsible for formatting the currencies is already tested.

File: src/javascript/app_2/Utils/URL/__tests__/url_helper.spec.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
+import { expect }   from 'chai';
+import URLHelper    from '../url_helper';
+
+describe('URLHelper', () => {
+    let url;
+    beforeEach('Setting up url and params', () => {
+        url = 'https://www.binary.com/en/trading.html?currency=USD&market=forex';
```
[the remark this answers is not in the document]
Comment (ahkuma) [reply src/javascript/app_2/Utils/URL/__tests__/url_helper.spec.js]: Because it doesn't return an URL.

**Pull request merged**